src/Resource/Otp.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace SMSGlobal\Resource;
+
+use Cassandra\Date;
+use SMSGlobal\Exceptions\AuthenticationException;
+use SMSGlobal\Exceptions\InvalidPayloadException;
+use SMSGlobal\Exceptions\InvalidResponseException;
+use SMSGlobal\Exceptions\ResourceNotFoundException;
+
+/**
+ * Class Otp
+ * @package SMSGlobal\Resource
+ */
+class Otp extends Base
+{
+    /**
+     * @var string
+     */
+    protected $resourceUri = '/otp';
+
+    /**
+     * @param string $to
+     * @param string $text
+     * @param string|null $from
+     * @param int $codeExpirt
+     * @param int $length
+     * @param \DateTime|null $messageExpiryDateTime
+     *
+     * @return array
+     * @throws AuthenticationException
+     * @throws GuzzleException
+     * @throws InvalidPayloadException
+     * @throws InvalidResponseException
+     * @throws ResourceNotFoundException
+     */
+    public function send(string $to, string $text, string $from = null, string $codeExpiry = null, string $length = null, \DateTime $messageExpiryDateTime = null): array
+    {
+        $origin = !empty($from) ? $from : '';
+        $codeExpiry = !empty($codeExpiry) ? $codeExpiry : '';
+        $length = !empty($length) ? $length : '';
+        $messageExpiryDateTime = !empty($messageExpiryDateTime) ? $messageExpiryDateTime->format(self::DATE_FORMAT) : '';
```

Comment:
These validations seem quite basic. Can we use the Symfony validator to do a better job?

Reply:
we already got fool proof validation on the api side so I dont feel any need of this.